### ISSUE TITLE
Maintenance Status of this Fork?

### ISSUE THREAD

**Issue by Gemba** [detain/skyscraper#25]
Hi @detain 

As this is the go-to-repo recommended by the original author of Skyskraper I would expect it to be more active. However, last commit/change is from Oct 2022 and looking at your GH activity timeline it seems your are not AFK.

If it would be more maintained there is a chance it could be integrated into the RetroPie project, which would bring you one step closer to world domination (your self-proclaimed target).

Thanks for a brief note about the future of this fork.

**Comment by maxexcloo** [comment]
Also curious about this, this project is awesome and I'd love to be able to use an up to date version :)

EDIT: It seems OP has forked the project and made quite a lot of new improvements: https://github.com/Gemba/skyscraper

**Comment by Gemba** [comment]
Yep. Since last autumn my fork is also approved as successor after Lars did discontinue the development on Skyscraper.
https://github.com/muldjord/skyscraper/#code-contributions-and-forks
